### ISSUE TITLE
CLAM workflow without test set

### ISSUE THREAD

**Issue by andrewsris** [mahmoodlab/CLAM#68]
Hello,

I am interested in using the CLAM workflow on a project of ours, but we currently do not have many slides (only about 26 slides). I am interested to see if CLAM can be helpful in a proof-of-concept framework, and I would really only be interested in a training and validation set, and not a test set at this moment. Is there a straightforward method to use CLAM without a test set? Or would this require rewriting many of the core scripts included in this package?

Thanks!

**Comment by fedshyvana** [comment]
Sure - if you want, as a hack, just create splits for train/val, and then add a test column that is a copy of the val (basically make the test set the same as validation). And if you are concerned with using early_stopping based on the validation loss when you're in fact using the validation set for testing - just turn off early_stopping.

**Comment by andrewsris** [comment]
Thank you very much! I will try this soon, but it sounds like it should work nicely.
